Demo/Demo/UseCases/GetOrderIdUseCase.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  GetOrderIdUseCase.swift
+//  Demo
+//
+//  Created by Jose Noriega on 31/08/2022.
+//
+
+import UIKit
+
+final class GetOrderIdUseCase {
+
+    func execute() async throws -> String {
+        let order = try await DemoMerchantAPI.sharedService.createOrder(order: GetOrderIdUseCase.orderWithFixedShipping)
+        return order.id
+    }
+
+    private static let orderWithFixedShipping = """
```

Comment:
Same comment here about using a `Codable` struct.

Reply:
Also happy to pair on this too!

Reply:
same! we can pair on this